### DISCUSSION TITLE
[Detector Support]: Coral temperature

### DISCUSSION THREAD

**Discussion by sebsebseb1982** [blakeblackshear/frigate#15510] [Detector Support]
### Describe the problem you are having

Hi, I'm using a coral M2 dual TPU in a mini n100 pc (g3 gmktec). It is working but I'm a bit surprised with the coral temperature. Without any detector enabled, the coral is at 75°c ! with some cameras activated it is at ~89°C

On the attached screenshot, I've disabled cameras detection around 7:20 AM and temperature does not decrease significantly. 

Is it normal ? Do you have similar temperatures ?
Maybe it comes from m2 ports that are stacked ?

### Version

0.14.0-da913d8

### Frigate config file

```yaml
detectors:
    coral:
      type: edgetpu
      device: pci

  detect:
    fps: 5
```

### docker-compose file or Docker CLI command

```yaml
None
```

### Relevant Frigate log output

```shell
None
```

### Install method

HassOS Addon

### Object Detector

Coral

### Screenshots of the Frigate UI's System metrics pages

![signal-2024-12-14-114719](https://github.com/user-attachments/assets/6853e6de-4f3a-44cd-9831-b44cb0c82402)

**Comment by NickM-27** [comment]
It's best to ensure there is good airflow. I added a heatsink to mine and the temps stay around 60° C

**Comment by sebsebseb1982** [comment]
As my m2 slots are stacked, I think it will never get good airflow nor heatsink :\
